inc/problem.class.php

```diff
@@ -647,72 +647,101 @@ static function showCentralList($start, $status = "process", $showgroupproblems
          return false;
       }
 
-      $search_users_id = " (`glpi_problems_users`.`users_id` = '".Session::getLoginUserID()."'
-                            AND `glpi_problems_users`.`type` = '".CommonITILActor::REQUESTER."') ";
-      $search_assign   = " (`glpi_problems_users`.`users_id` = '".Session::getLoginUserID()."'
-                            AND `glpi_problems_users`.`type` = '".CommonITILActor::ASSIGN."')";
-      $is_deleted      = " `glpi_problems`.`is_deleted` = 0 ";
+      $WHERE = [
+         'is_deleted' => 0
+      ];
+      $search_users_id = [
+         'glpi_problems_users.users_id'   => Session::getLoginUserID(),
+         'glpi_problems_users.type'       => CommonITILActor::REQUESTER
+      ];
+      $search_assign = [
+         'glpi_problems_users.users_id'   => Session::getLoginUserID(),
+         'glpi_problems_users.type'       => CommonITILActor::ASSIGN
+      ];
 
       if ($showgroupproblems) {
-         $search_users_id = " 0 = 1 ";
-         $search_assign   = " 0 = 1 ";
+         $search_users_id  = [false];
```

Comment:
`false` will not work, as it will be converted as an empty string in iterator
```suggestion
          $search_users_id  = [0];
```